find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
```

Comment:
Might be helpful to add code to cover the edge case of when a user gives an invalid folder path